rtrlib/rtr/packets.c

```diff
@@ -749,6 +749,9 @@ static int rtr_undo_update_spki_table(struct rtr_socket *rtr_socket, void *pdu)
 
 /*
  * @brief Appends the Prefix PDU pdu to ary.
+ *
+ * @retval RTR_SUCCESS On success
+ * @retval RTR_ERROR On realloc failure, ary is not freed in this case, because it might contain data that is still needed
```

Comment:
for sake of consistency use `@return` here, like in the rest of the docu.

Though, we might reconsider the use of `@retval` and `@return` for docu, in general, but that would be another PR.